CHANGELOG.md

```diff
@@ -4,10 +4,16 @@
 
 ## Important Notes
 
+- [#624](https://github.com/oauth2-proxy/oauth2-proxy/pull/624) Add flag to strip authentication headers from whitelisted requests
+  - This is controlled with the `--skip-auth-strip-headers` flag (default `false`). The next major release will switch this to a more secure `true` default.
+  - `--pass-basic-auth` will now only control setting the `Authorization` header with basic auth. It no longer controls the `X-Forwarded-*` user headers (already configured with `--pass-user-headers`)
```

Comment:
This still counts as a breaking change doesn't it?

Reply:
Yeah true -- I'll revert it and rebase to get some cleaner commits.  I think we must've decided on these changes just before V6, but missed out breaking changes opportunity.

Reply:
I think so, sorry for the extra work

Reply:
Not a problem! All set -- this PR has the breaking changes and this one is now non-breaking: https://github.com/oauth2-proxy/oauth2-proxy/pull/679